modules/ROOT/partials/configuration/ai_request.adoc

```diff
@@ -0,0 +1,176 @@
+[[ai_request]]
+== `ai_request`
+
+The {pluginname} uses the `+ai_request+` function to send prompts to an AI endpoint, and display the responses.
+
+The `+ai_request+` function will be called each time a user submits a prompt. 
+
+These prompts are only submitted with the {pluginname} dialog open, whether from typing in the dialog input field, or from using an {pluginname} shortcut.
+
+The content returned within the `+ai_request+` function is displayed within the dialog, once a response is provided.
+
+NOTE: This option is required to use the {pluginname} plugin.
+
+*Type:* `+Function+`
+
+=== Example: using `ai_request` and the `stream` callback to interface with the OpenAI Completions API
+
+[source,js]
+----
+const fetchApi = import("https://unpkg.com/@microsoft/fetch-event-source@2.0.1/lib/esm/index.js").then(module => module.fetchEventSource);
+
+const api_key = '<INSERT_API_KEY_HERE>'
```

Comment:
```suggestion
const api_key = '<INSERT_API_KEY_HERE>';
```
Semi-colon added to end the statement as per @ShiridiGandham’s [comment](https://github.com/tinymce/tinymce-docs/pull/2819/files#r1267643527).